opentimelineview/timeline_widget.py

```diff
@@ -205,6 +214,67 @@ def __init__(self, *args, **kwargs):
         self.source_name_label.setText('GAP')
 
 
+class Effect(QtWidgets.QGraphicsRectItem):
+
+    def __init__(self, item, rect, *args, **kwargs):
+        super(Effect, self).__init__(rect, *args, **kwargs)
+        self.item = item
+        self.setFlags(QtWidgets.QGraphicsItem.ItemIsSelectable)
+        self.init()
+        self._set_tooltip()
+
+    def init(self):
+        rect = self.rect()
+        rect.setY(TRACK_HEIGHT - EFFECT_HEIGHT)
+        rect.setHeight(EFFECT_HEIGHT)
+        self.setRect(rect)
+
+        dark = QtGui.QColor(0, 0, 0, 150)
+        colour = QtGui.QColor(255, 255, 255, 200)
+        gradient = QtGui.QLinearGradient(
+                   QtCore.QPointF(0,
+                                  self.boundingRect().top()),
+                   QtCore.QPointF(0,
+                                  self.boundingRect().bottom()))
+        gradient.setColorAt(0.2, QtCore.Qt.transparent)
+        gradient.setColorAt(0.45, colour)
+        gradient.setColorAt(0.7, QtCore.Qt.transparent)
+        gradient.setColorAt(1.0, dark)
+        self.setBrush(QtGui.QBrush(gradient))
+
+        pen = self.pen()
+        pen.setColor(QtGui.QColor(0, 0, 0, 80))
+        pen.setWidth(0)
+        self.setPen(pen)
+
+    def _set_tooltip(self):
+        tool_tips = list()
+        for effect in self.item:
+            name = effect.name if effect.name else ""
+            effect_name = effect.effect_name if effect.effect_name else ""
+            tool_tips.append("%s %s" % (name, effect_name))
```

Comment:
We're trying to use the newer (and python3 friendlier) `.format` style of string formatting in python:

```python
"{} {}".format(name, effect_name)
```